eleventy.config.js

```diff
@@ -84,6 +87,6 @@ module.exports = function (eleventyConfig) {
             input: "src"
         },
         passthroughFileCopy: true,
-        markdownTemplateEngine: "njk"
+        markdownTemplateEngine: "webc"
```

Comment:
This was a change specific to the Guide and should not be made here. It has nothing to do with Decap CMS.